exporter/metrics.py

```diff
@@ -0,0 +1,36 @@
+from prometheus_client import Histogram, Gauge, CollectorRegistry
+
+registry = CollectorRegistry()
+
+TasksByStateGauge = Gauge(
+    "celery_tasks", "Number of tasks per state", ["state"], registry=registry
+)
+
+TasksByStateAndNameGauge = Gauge(
+    "celery_tasks_by_name",
+    "Number of tasks per state and name",
+    ["state", "name"],
+    registry=registry,
+)
+
+TaskRuntimeByNameHistogram = Histogram(
+    "celery_tasks_runtime_seconds",
+    "Task runtime (seconds)",
+    ["name"],
+    registry=registry,
+)
+
+WorkersGauge = Gauge("celery_workers", "Number of alive workers", registry=registry)
+
+# TaskLatencyHistogram = Histogram(
+#     "celery_task_latency",
+#     "Seconds between a task is received and started.",
+#     registry=registry,
+# )
```

Comment:
Is there any reason we're omitting this? Currently we have a heatmap for it (and might be worth adding an alert for it in the future)
https://mon.partnerstack.io/d/zXb9ma6Wk/worker-v2?orgId=1

Reply:
I can add it back in, its in the README.md under Roadmap

Reply:
Just wanted to get the basics working in this first pass